### PR TITLE
reword "All Conformant Kubernetes" to "All Certified Kubernetes Distributions"

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR rewords "All Conformant Kubernetes" to "All Certified Kubernetes Distributions".

This is a subtle but important change for inclusivity, as I have seen many users not understanding what "Conformant" means (especially people for whom English is not their first language).

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@jbottum @zijianjoy As this only affects "Charmed Kubeflow" and my distributions (deployKF, Argoflow), I think we only need approval from Canonical.

@DomFleischmann can you confirm from the Canonical side?

**Comment by jbottum** [comment]
For Canonical, I would differ to @DnPlas and @kimwnasptd.    I think this change is for the distributions to make individually.   Who is the contact for ArgoFlow and are they engaged ?   Shouldn't Arrikto be removed from the table?...I thought that was already done.

**Comment by thesuperzapper** [comment]
@jbottum that was a lot of questions in one sentence:

1. Sounds good, if the current Canonical reps are  @DnPlas and @kimwnasptd, they can confirm they prefer this wording
2. Argoflow was created by @DavidSpek and me, but it is [now archived](https://github.com/argoflow/argoflow), and I think we might as well use the new wording to reduce confusion and keep everything aligned
3. If the "Arrikto Enterprise Kubeflow" is no longer a thing, I will raise a separate PR for that:
     - _NOTE: we removed the "Arrikto Kubeflow as a Service" in https://github.com/kubeflow/website/pull/3549, which was a separate entry_

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by DnPlas** [comment]
ACK, I need to confirm that stating "all distributions of kubernetes" actually reflects what Charmed Kubeflow supports before we can approve this PR.

**Comment by thesuperzapper** [comment]
There was a missing HTML tag, so I have fixed that in https://github.com/kubeflow/website/pull/3573/commits/30b40f6fb85a7da8d076a6dd24fb688074868614

**Comment by thesuperzapper** [comment]
@DnPlas I was hoping you could confirm that Canonical is OK with this minor change that improves the wording on the website listing of the Charmed Kubeflow distribution.

**Comment by thesuperzapper** [comment]
@DnPlas I think `Official CNCF Kubernetes distributions` is not valid, because there are no "official" CNCF distributions.

However, there are [CNCF "certified" distributions](https://www.cncf.io/certification/software-conformance/), which is a much easier word for people to understand.

So perhaps we could use `All Certified Kubernetes Distributions`?

File: content/en/docs/started/installing-kubeflow.md
Comment (chensun) [inline content/en/docs/started/installing-kubeflow.md:132]: Certified by whom? Any references?

Comment (thesuperzapper) [reply content/en/docs/started/installing-kubeflow.md:132]: @chensun I am referring to the CNCF Kubernetes Certification program: 
- https://www.cncf.io/certification/software-conformance/

I can add that as a link if you like?

Comment (chensun) [reply content/en/docs/started/installing-kubeflow.md:132]: That would be ideal.

Comment (thesuperzapper) [reply content/en/docs/started/installing-kubeflow.md]: @chensun done. 

I have linked to the official list on the Kubernetes website: https://kubernetes.io/partners/#conformance

**Review by chensun — APPROVED** [review]
/lgtm

**Pull request merged**
